### PR TITLE
Add language reference section to README

### PR DESCRIPTION
**Pull request by lpil**
#56

I think it might be nice to later extract this into a little website like Jade does http://jade-lang.com/reference/

Perhaps we could make a github organisation for this project and host it with github pages.
i.e. github.com/slim**e**-template

### CODE REVIEW

**Comment by doomspork** [comment]
:boom: Awesome work @lpil.  I'll start looking this over and let you know if I have any feedback.  cc @henrik, @Rakoth

Good suggestion RE: github pages.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
+[hex-img]: https://img.shields.io/hexpm/v/slim_fast.svg
+[hex]: https://hex.pm/packages/slim_fast
+[license-img]: http://img.shields.io/badge/license-MIT-brightgreen.svg
+[license]: http://opensource.org/licenses/MIT
 
 > A refreshing way to slim down your markup in Elixir.
 
-SlimFast is an [Elixir](http://elixir-lang.com) library for rendering [Slim](http://slim-lang.com) templates as HTML.
+SlimFast is an [Elixir](http://elixir-lang.com) library for rendering
+[Slim](http://slim-lang.com)-like templates as HTML.
```
Comment (doomspork) [inline README.md]: Let's remove the Phoenix section now that it's belong the reference and update this to something like:  

"SlimFast is an [Elixir](elixir) library for rendering [Slim](slim)-like templates as HTML.  For use with [Phoenix](http://www.phoenixframework.org), please see [PhoenixSlim](https://github.com/doomspork/phoenix_slim)."

Thoughts?

Thanks again @lpil :+1:

**Comment by lpil** [comment]
There you go. :)

**Comment by doomspork** [comment]
Looks good, thank you again @lpil :+1:

**Pull request merged**